### DISCUSSION TITLE
MASQUE出站连接Cloudflare WARP

### DISCUSSION THREAD

**Discussion by hamjin** [XTLS/Xray-core#3843] [Ideas]
https://blog.cloudflare.com/zero-trust-warp-with-a-masque/
如题，不太想用cloudflare-warp开SOCKS5代理

**Comment by wulfric93** [comment]
MASQUE protol is indeed a breakthrough !

**Comment by RememberOurPromise** [comment]
https://github.com/Diniboy1123/usque/
